@AH-64D Apache Longbow/Addons/fza_ah64_systems/cfgVehicles/hitPoints.hpp

```diff
@@ -338,7 +338,7 @@ class HitPoints
         radius = HIT_RADIUS_01;
         minimalHit = MIN_HIT;
         explosionShielding = 0.80;
-        name = "hitvrotor";
+        name = "";
```

Comment:
why was this removed? it's not removed from hithrotor

Reply:
Accidental